src/NetscriptFunctions/Singularity.ts

```diff
@@ -133,18 +133,18 @@ export function NetscriptSingularity(player: IPlayer, workerScript: WorkerScript
         return aug.prereqs.slice();
       },
     getAugmentationPrice: (_ctx: NetscriptContext) =>
-      function (_augName: unknown): number {
+      function (_augName: unknown, _basePrice: unknown = false): number {
         _ctx.helper.checkSingularityAccess();
         const augName = _ctx.helper.string("augName", _augName);
         const aug = getAugmentation(_ctx, augName);
-        return aug.getCost(player).moneyCost;
+        return !_basePrice ? aug.getCost(player).moneyCost : aug.baseCost;
```

Comment:
dont forget to bump the bool through the helper, although i wonder how useful the base price would be considering the sleeve function exists

`const returnBasePrice = _ctx.helper.boolean("returnBasePrice", _returnBasePrice);`

Reply:
imo it makes sense to me to keep it clean and separate. the basePrice outside of sleeves is essentially source code at this point imo